src/test/java/gov/healthit/chpl/aqa/stepDefinitions/LoginSteps.java

```diff
@@ -46,7 +50,7 @@ public void loginFormShowsInDropdown() throws Throwable {
 
     @Given("^User enters valid \"(.*)\" and \"(.*)\"$")
     public void userEntersValidUsernameAndPassword(final String username, final String password) throws Throwable {
-        driver.get("https://chpl.ahrqdev.org/#/resources/download");
+        driver.get(url + "#/resources/download");
```

Comment:
Shouldn't change pages in a step that doesn't say anything about changing pages

Reply:
Login feature needs to be re-written (it was one of the initial scripts before setting up page objects). I'd have to request new ids/objects and add a new page under page objects just for login. That way, I can reuse login feature in other step definitions, wherever applicable. I am going to remove current step definition file.